renku/core/management/migrations/m_0003__1_jsonld.py

```diff
@@ -15,13 +15,50 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Migrations for dataset."""
+"""JSON-LD dataset migrations."""
+import itertools
 import os
 import uuid
 from pathlib import Path
 
+from renku.core.models.datasets import Dataset
 
-def migrate_dataset_schema(data):
+__all__ = [
+    'migrate_datasets_metadata',
+]
+
+
+def migrate_datasets_metadata(client):
+    """Apply all initial JSON-LD migrations to datasets."""
+    jsonld_migrations = {
```

Comment:
The initial migration is missing the `translate` on-the-fly migrations we have at the moment, see https://github.com/SwissDataScienceCenter/renku-python/blob/master/renku/core/models/projects.py#L42 for an example.